wix/Includes/OpenJDK.Variables.wxi

```diff
@@ -11,13 +11,13 @@
     <?define Arch="(x64)" ?>
   <?endif?>
 
-  <?define ProductVersion="$(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion)" ?>
+  <?define ProductVersion="$(var.MSIProductVersion)" ?>
   <?define ProductLanguage="!(loc.ProductLanguage)" ?>
   <?define ProductManufacturer="AdoptOpenJDK" ?>
 
   <?if $(env.PRODUCT_CATEGORY)="jre"?>
     <?define ProductName="!(loc.JREProductName)" ?>
-    <?define ProductNameWithVersion="!(loc.JREProductName) $(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion) $(var.Arch)" ?>
+    <?define ProductNameWithVersion="!(loc.JREProductName) $(var.ProductFullVersion) $(var.Arch)" ?>
```

Comment:
Should that be ProductVersion or ProductFullVersion?

Reply:
ProductFullVersion is the name of the var I passed in from the .cmd file.

Reply:
-dProductFullVersion="!JEP322_NAME!"

Reply:
As it is only a desc we prefer the longer version name and not the computed one with *100 hack